File: OnlineJudge/p10041.py
import sys

def vito_house(test_cases):
    results = []
    for case in test_cases:
        relatives = sorted(case)
        median = relatives[len(relatives) // 2]
        total_distance = sum(abs(r - median) for r in relatives)
        results.append(total_distance)
    return results

def main():
    input = sys.stdin.read
    data = input().splitlines()
    test_cases = []
    n = int(data[0])
    for i in range(1, n + 1):
        line = list(map(int, data[i].split()))
        test_cases.append(line[1:])
    results = vito_house(test_cases)
    for res in results:
        print(res)

if __name__ == "__main__":
    main()

File: OnlineJudge/p10041.cpp
#include <iostream>
#include <vector>
#include <algorithm>

using namespace std;

int main()
{
    int case_size;
    cin >> case_size;

    while (case_size--)
    {
        int r;
        cin >> r;

        vector<int> streets(r);
        for (int i = 0; i < r; ++i)
        {
            cin >> streets[i];
        }

        sort(streets.begin(), streets.end());
        int median = streets[(int)(r / 2)]; 
        int total_distance = 0;
        
        for(int i=0;i<streets.size();i++)
        {
            total_distance += abs(streets[i] - median);
        }

        cout << total_distance << endl;
    }

    return 0;
}